pkgs/haskell/ghcup-hs/registry.yaml

```diff
@@ -0,0 +1,15 @@
+packages:
+- type: http
+  repo_owner: haskell
+  repo_name: ghcup-hs
```

Comment:
Add `repo_owner` and `repo_name` for getting the version from GitHub Releases in `aqua g`.